### PR TITLE
manolo tlc ciel d5 5_1d to 5_2_1

### PR DESCRIPTION
**Pull request by nolo16**

### CODE REVIEW

File: script/Ciel/Day 5/05_01_CIEL05_1D.txt [hunk 1]
```diff
@@ -84,14 +83,13 @@ In such a place... I couldn't even finish my question before I'm pinned to the w
...
-Why are you so calm...?!"
+Why are %{i}you%{/i} so calm...?!"
 }
 [sha:6b9da94e458916dc624de1a2880338c64546d65c]{
 -- Page 158, Offset 11301.
 -- がん、と胸元を<拳骨|げんこつ>の底で叩かれる。
-// If anyone has a better idea of what 拳骨の底 is...
-Thud. She hits me on the chest with the heel of her palm.
+// If anyone has a better idea of what 拳骨の底 is... lmao.... idfk
+Thud. She hits me on the chest with the bottom of her fist.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_1D.txt:92]: base of her fist?? maybe??

File: script/Ciel/Day 5/05_01_CIEL05_1_1.txt [hunk 2]
```diff
@@ -88,6 +88,6 @@ The foul sound of him swallowing resonates in my ears and in my brain.
 [sha:0284ebdabdf4a1ded03acd8be448c7685c5d1181]{
 -- Page 276, Offset 11597.
 -- ―――おぞましい動く死人。ここで、死ぬまで殺してやる。
-// Lit. 死ぬまで殺してやる "I"ll kill you until you die/you're dead". I thought about "until you die/you're dead for real", or using "I'll kill you dead"
+// Lit. 死ぬまで殺してやる "I"ll kill you until you die/you're dead". I thought about "until you die/you're dead for real", or using "I'll kill you dead" i personally dont mind "I'll kill you dead"
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_1_1.txt:91]: well I can't say I don't mind it, but I see it as pretty acceptable

File: script/Ciel/Day 5/05_01_CIEL05_1_1B.txt [hunk 3]
```diff
@@ -12,6 +12,7 @@ Vlov is unaware of my presence.
 -- Page 10008, Offset 11600.
 -- 女との情事の最中であり、
 // 情事 a love affair, a liaison
+// nasu.......
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_1_1B.txt:15]: bruh

File: script/Ciel/Day 5/05_01_CIEL05_1E.txt [hunk 4]
```diff
@@ -21,13 +21,14 @@ The ceiling is supported by numerous pillars.
 [sha:b027beb5f34b748e687fc98331aac8d9992c0e32]{
 -- Page 199, Offset 11399.
 -- 天井中心部には窓らしき穴があり、灰色の陽差しを下ろしている。それこそ、雲の隙間からのぞく<天使の梯子|ジェイコブズラダー>のように。
-In the center of the ceiling, there's a window-like hole from which the greyish sunlight fell. It's exactly like a <sunbeam|J a c o b ' s  l a d d e r> peeking through gaps in the clouds.
+// what the shit is jacobs ladder
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_1E.txt:24]: this is a double whammy of Nasu's beloved superficial comparative theology crap and some wacky foreign term to trip up the native readers. [Jacob's Ladder](https://www.koco.com/article/rays-of-sunlight-known-as-jacob-s-ladder/4300753) is both a phenomenon where light shines through clouds and a biblical allusion

File: script/Ciel/Day 5/05_01_CIEL05_1E.txt [hunk 5]
```diff
@@ -243,17 +245,18 @@ I realize the unsightliness of running away from my guilt, of the selfishness of
...
 -- Page 217, Offset 11444.
 -- でも。その上で、俺は俺にできる事をしたいんだ。
-But. I also want to do what I can.
+But, I also want to do whatever I can.
 }
 [sha:75768039b970fb92daf44ac9719cac4a6628b763]{
 -- Page 217, Offset 11445.
 -- ここで逃げ出したら本当に、俺は<無|た><能|だ>の人殺しだ。そんなヤツに、罪の償いができる筈がない。
-If I run away here, then I'll really be <an incompetent|j u s t  a  > murderer. There's no way I'll be able to atone for my sins then.
+// not too sure, can bring back ruby if this one doesnt read well
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_1E.txt:258]: I'm kinda indifferent. I don't really think the ruby adds anything and it kinda makes it confusing to read

File: script/Ciel/Day 5/05_01_CIEL05_1E.txt [hunk 6]
```diff
@@ -339,17 +342,17 @@ Having run as fast as I could, I rest my body against the platform.
 [sha:0fc038acffed903f2775f47d3b2ec14631f47c97]{
 -- Page 224, Offset 11462.
 -- 自分の意思による明確な殺害行為。その嘔吐感をかみ砕く。
-It was a clear act of murder, done with my own volition. I hold back the urge to vomit.
+It was a clear act of murder, done at my own volition. I hold back the urge to vomit.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_1E.txt:345]: of my own volition

File: script/Ciel/Day 5/05_01_CIEL05_1E.txt [hunk 7]
```diff
@@ -359,13 +362,14 @@ I don't have the time to turn around.
...
 -- 寝台に眠らされた、見知らぬ女生徒を抱き起こす。
-I lift in my arms the unfamiliar schoolgirl that was put to sleep on the bed.
+I lift the unfamiliar school girl who was laid to sleep on the platform in my arms.
 }
 [sha:442294158e2af81c47df260ee3ed8d6bd9b08176]{
 -- Page 226, Offset 11467.
 -- 乱暴だが叩いてでも目を覚ましてもらわないと、さすがにここから逃げられ―――
 // Someone check this
-It's rough but if she doesn't wake up even if I knock her around, we'll never be able to escape from he―――
+// barbaric? violent?
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_1E.txt:371]: barbaric is fine

File: script/Ciel/Day 5/05_01_CIEL05_1E.txt [hunk 8]
```diff
@@ -490,13 +494,13 @@ I recognize it as the body of one of the zombies standing near the entrance.
...
 -- 先生と呼んでいた女性の、とんでもない姿を目の当たりにした。
-I saw right in front of my eyes the preposterous appearance of the woman who I call teacher.
+Before my very eyes is a preposterous sight; the outrageous figure of the woman I call teacher.
 }
 [sha:9533d4fdd63cf6cfee57ceb605284418261798c5]{
 -- Page 238, Offset 11493.
 -- 「おとなしく潜入捜査やってれば調子に乗ってくれちゃって。
-// Need double check
-"Just because I'm investigating undercover, they're getting cocky!
+// Need double check looks fine?
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_1E.txt:502]: looks fine to me

File: script/Ciel/Day 5/05_01_CIEL05_1E.txt [hunk 9]
```diff
@@ -734,13 +738,13 @@ I just realized.
...
 // Added engraved
-The blood from the crushed corpses was flowing down into pattern engraved in the floor.
+The blood from the crushed corpses was flowing down into a pattern engraved in the floor.
 }
 [sha:a96a832932be27f9374512e447cf42c5fc9436b2]{
 -- Page 258, Offset 11541.
 -- それは何かの儀式のように、<よ|・><く|・><な|・><い|・><モ|・><ノ|・>を呼び寄せている。
-// Something bad in italics, ruby dots <よ|・><く|・><な|・><い|・><モ|・><ノ|・>
-Like some sort of ritual that would summon something bad.
+// is dreadful too extreme here?
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_1E.txt:746]: for よくない... probably a bit strong. I'd probably just go for like "unpleasant"

**Review by Avuxo — APPROVED** [review]
お疲れさまでした　
image

File: script/Ciel/Day 5/05_01_CIEL05_1D.txt [hunk 10]
```diff
@@ -53,18 +52,18 @@ I actually want to curl up inside that hidden place, but I have to keep my compo
...
 -- Page 153, Offset 11295.
 -- こんな場所にいるのか、と訊くより早く、壁に押しつけられていた。
-In such a place... I couldn't even finish my question before I'm pinned to the wall.
+In such a place... I couldn't even finish my question before I was pinned to the wall.
 }
 [sha:9fc8335cc88883c9a5a7f61ab1d9c5392e2a4e20]{
 -- Page 154, Offset 11296.
 -- 「いやよ、どうして安全じゃないの!?　安全にしてよ！　わたしは生徒指導に来ただけ！　女の子限定のあやしいアルバイトなんて聞いたらほっとけないでしょう!?　残業なんてガラじゃなかったけど、たまたま、たまたま目の前でうちの子が連れていかれちゃったら放っておけないじゃない……！」
-"Noo, why isn't it safe?! Make it safe! I just came as a student counselor. When I heard about a suspicious part-time job exclusively for girls, I had to get involved, right?! It's not like me to put in extra hours, but if by chance, just by chance, I see a girl I'm responsible for get taken away in front of me, I can't leave them alone...!"
+// noel lines give me a fucking aneurysm
```
[the remark this answers is not in the document]
Comment (nolo16) [reply script/Ciel/Day 5/05_01_CIEL05_1D.txt:65]: nasu is projecting

**Pull request merged**